cdc/capture/http_handler.go

```diff
@@ -576,16 +576,19 @@ func (h *HTTPHandler) GetProcessor(c *gin.Context) {
 	status, exist := statuses[captureID]
 	if !exist {
 		_ = c.Error(cerror.ErrCaptureNotExist.GenWithStackByArgs(captureID))
+		return
 	}
 
 	positions, err := statusProvider.GetTaskPositions(ctx, changefeedID)
 	if err != nil {
 		_ = c.Error(err)
 		return
 	}
+
 	position, exist := positions[captureID]
 	if !exist {
 		_ = c.Error(cerror.ErrCaptureNotExist.GenWithStackByArgs(captureID))
+		return
```

Comment:
the API of `gin` is too ugly, i still can't understand why it can't return error 😂

Reply:
I really agree with you in this point.